testutil/ciao-down/ciao_down.go

```diff
@@ -144,32 +144,41 @@ func checkDirectory(dir string) error {
 	return nil
 }
 
-func prepareFlags(ws *workspace) (*workload, bool, error) {
+func createFlags(ws *workspace) (*workload, bool, error) {
 	var debug bool
-	var vmType string
 	var m mounts
 	var p ports
 	var memGiB, CPUs int
 	var update packageUpgrade
 
-	fs := flag.NewFlagSet("prepare", flag.ExitOnError)
+	fs := flag.NewFlagSet("create", flag.ExitOnError)
+	fs.Usage = func() {
+		fmt.Fprintf(os.Stderr, "Usage: %s create <workload> \n\n", os.Args[0])
+		fmt.Fprintf(os.Stderr, "  <workload>\tName of the workload to create\n\n")
+		fs.PrintDefaults()
+	}
 	vmFlags(fs, &memGiB, &CPUs, &m, &p)
 	fs.BoolVar(&debug, "debug", false, "Enables debug mode")
-	fs.StringVar(&vmType, "vmtype", CIAO, "Type of VM to launch.")
 	fs.Var(&update, "package-upgrade",
 		"Hint to enable or disable update of VM packages.  Should be true or false")
 
 	if err := fs.Parse(flag.Args()[1:]); err != nil {
 		return nil, false, err
 	}
 
+	if fs.NArg() != 1 {
+		fs.Usage()
+		os.Exit(1)
```

Comment:
Better to return an error here rather than exit straight away as by exiting we're by-passing ciao-down's normal shutdown sequence.  The problem with exiting is that your defers don't get called so someone adding a defer to this function or a function further up in the callstack would be surprised to discover that their deferred function does not get called.